src/js/stores/DCOSStore.js

```diff
@@ -402,6 +409,23 @@ class DCOSStore extends EventEmitter {
    * @type {ServiceTree}
    */
   get serviceTree() {
+    if (!this._serviceTree) {
+      this.serviceTree = this.buildServiceTree();
+    }
+
+    return this._serviceTree;
+  }
+
+  set serviceTree(newValue) {
+    this._serviceTree = newValue;
+    this._flatServiceTree = null;
```

Comment:
Same as on master. `this._flatServiceTree` is not being used. Maybe we should just delete it